pkg/apis/v1alpha1/stage_types.go

```diff
@@ -118,6 +118,20 @@ type StageNext struct {
 	Delete bool `json:"delete,omitempty"`
 	// StatusTemplate indicates the template for modifying the status of the resource in the next.
 	StatusTemplate string `json:"statusTemplate,omitempty"`
+	// StatusSubresource indicates the name of the subresource that will be patched. A nil value means
+	// entire object will be patches
+	// +kubebuilder:default=status
```

Comment:
```suggestion
	// +default="status"
	// +kubebuilder:default=status
```

The `+default="status"` for zz_generated.defaults.go


use ` ./hack/update-codegen.sh` to regenerate

Reply:
done